README.md

```diff
@@ -122,6 +122,15 @@ Here is the gif of the complete process when you execute the Upload command (Mig
 
 ![Upload](https://media.giphy.com/media/xT9IglKxSqs2Wdwq2c/source.gif)
 
+### Troubleshooting
+
+If for some reason you get an error when following these instructions try the following:

```

Comment:
public Wiki edit is enabled. What i suggest to make readme more extensive. 
We can give a link of public wiki in read me, and put all the possibilities there.

Reply:
You can create new page and put these things there and add link of troubleshooting here.

Reply:
Done! https://github.com/shanalikhan/code-settings-sync/wiki/Troubleshooting#invalid-gist-id-or-invalid--expired-github-token

Feel free to close this PR if you'd like, or I will update it to point to the wikipage if you'd prefer.